external-api/getting-started.md

```diff
@@ -6,9 +6,10 @@ API.
 
 ## Determining User Keys
 
-You'll want to request a user key from each affiliate you would like to make bookings on behalf of.
-You'll use this key for each request you make on behalf of the associated affiliate.
-Alternatively, contact <support@fareharbor.com> for help identifying the relevant user keys.
+FareHarbor will provide approved API partners with keys that can be used to make requests and create bookings for suppliers.
+The supplier must confirm their approval of your API access before you are able to make bookings for them via API.
+If you are using the FareHarbor API to book for suppliers across multiple countries/currencies, you may be provided with one User Key per currency.
+You can find more information about the authentication process in `authentication.md`. Alternatively, you may contact <support@fareharbor.com> for assistance with User Keys.
```

Comment:
Add link to authentication.md so it's easier to go to that doc while reading "getting-started.md"